src/components/Lesson/Lesson.js

```diff
@@ -171,40 +191,48 @@ const Lesson = React.createClass({
 });
 
 Lesson.propTypes = {
+  // ownProps
+  path: PropTypes.string,
   params: PropTypes.shape({
     file: PropTypes.string.isRequired
   }).isRequired,
   lesson: PropTypes.shape({
     frontmatter: PropTypes.object,
     content: PropTypes.string
```

Comment:
Lesson har f.eks. getTitle, en funksjon inni componenten, som henter ut this.props.lesson.frontmatter.title. Jeg tenker det hadde vært mer fornuftig at mapStateToProps hentet ut dette, istedet for å bare servere hele lesson.